aten/src/ATen/test/CMakeLists.txt

```diff
@@ -21,6 +21,10 @@ list(APPEND ATen_CPU_TEST_SRCS
   ${CMAKE_CURRENT_SOURCE_DIR}/verify_api_visibility.cpp
   ${CMAKE_CURRENT_SOURCE_DIR}/tbb_init_test.cpp
   ${CMAKE_CURRENT_SOURCE_DIR}/weakref_test.cpp)
+if (NOT USE_ROCM)
```

Comment:
Sorry but why is it related to ROCM?

Reply:
It's not as far as I can tell, I just took it from https://github.com/pytorch/pytorch/pull/15153. Should I remove it?

Reply:
@asuhan I see, I'm now asking in the original PR https://github.com/pytorch/pytorch/pull/15153/files#r252502306

Reply:
Removed the guard.